paper.md

```diff
@@ -0,0 +1,74 @@
+
+CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different options to decide on the best way to provide sustainable, affordable and reliable electricity to the community. Its simulation functionality offers the user an insight into the hourly, daily, seasonal and multi-year technical performance of a system in response to changing electricity demands and varying renewable resources. Its optimisation process replicates the design process common in sustainable development settings by identifying optimum system design over short-term time horizons, but allowing the system to increase in capacity over its lifetime as the demand of the community grows in response to its economic development [@Sandwell:2017a].
+
+# Main features
+
+## Electricity supply
+CLOVER can simulate a variety of electricity supply technologies:
+- Solar generation is sourced from the Renewables.ninja API [@Renewables_Ninja] with the synthesis of the data described in @Pfenninger:2016;
+- Battery storage, charged from solar, with user-defined characteristics for battery performance and lifetime to emulate many kinds of technologies;
+- Diesel generation can be included as a source of power when no other source is available;
+- Grid power can be treated as a primary or backup source of electricity with a user-defined availability profile.
+These electricity sources can be included as the single source of electricity or in any combination as part of a hybrid system.
+
+## Electricity demand
+Users can generate electricity load profiles from the bottom up. Load profiles are based on the probability that individual user-specified appliances are in use, which can vary throughout the day and year, using processes described in @Sandwell:2016 and applied in @Chambon:2020. Any kind of electrical appliance and usage pattern can be defined by the user and ownership of appliances can be set to increase over time. Demands can be segregated into domestic, commercial, public, or other demand categories and included independently within the system. Alternatively users can provide their own hourly load profile, for example taken from monitored data.
+
+## Simulation
+CLOVER assesses the balance of energy sources and demands at an hourly resolution over a user-specified lifetime period. During this process data including energy flows, battery state of charge, demand satisfaction, and service availability are recorded.
```

Comment:
Add commas around the intermediary clause:
During this process data**,** including energy flows, battery state of charge, demand satisfaction, and service availability**,** are recorded.

Reply:
Included as suggested